universum.cabal

```diff
@@ -13,9 +13,10 @@ category:            Prelude
 stability:           stable
 build-type:          Simple
 bug-reports:         https://github.com/serokell/universum/issues
-tested-with:         GHC == 8.4.3
+tested-with:         GHC == 8.4.4
                    , GHC == 8.6.5
-                   , GHC == 8.8.1
+                   , GHC == 8.8.3
+                   , GHC == 8.10.1
```

Comment:
Maybe mention in CI config file that when the respective dimension of the build matrix is updated, cabal file has to be updated as well? Otherwise, 80% that it would be forgotten if done by someone first time.

Reply:
Done.